test/Service/AuthorizationServiceTest.php

```diff
@@ -157,15 +160,15 @@ public function testReturnsTrueForIdentityWhenHasPermissionButNoAssertionsExists
 
     public function testUsesAssertionsAsInstances()
     {
-        $role = new FlatRole('admin');
-        $identity = new Identity([$role]);
+        $role      = new FlatRole('admin');
```

Comment:
Well, maybe that's me but I find it more confortable to have keys and values aligned. Just personal feeling :)